### ISSUE TITLE
Translation assistance

### ISSUE THREAD

**Issue by gibsonc** [FPGAwars/Collection-Jedi#6]
I would like to assist with the translation of Collections, however, no matter what I do, or what locale codes I use, the text remains that which is in the base package. Is translation support implemented on Collections?

**Comment by Jesus89** [comment]
Yes, you can use this basic tool to manage the collection translations: https://github.com/FPGAwars/icm
